openstack/networking/v2/extensions/lbaas_v2/loadbalancers/requests.go

```diff
@@ -0,0 +1,197 @@
+	if opts.VipSubnetID == "" {
+		res.Err = errVipSubnetIDRequried
+		return res
+	}
+
+	type loadbalancer struct {
+		Name         *string `json:"name,omitempty"`
+		Description  *string `json:"description,omitempty"`
+		VipSubnetID  string  `json:"vip_subnet_id"`
+		TenantID     *string `json:"tenant_id,omitempty"`
+		VipAddress   *string `json:"vip_address,omitempty"`
+		AdminStateUp *bool   `json:"admin_state_up,omitempty"`
+		Flavor       *string `json:"flavor,omitempty"`
+		Provider     *string `json:"provider,omitempty"`
+	}
+
+	type request struct {
+		Loadbalancer loadbalancer `json:"loadbalancer"`
+	}
+
+	reqBody := request{Loadbalancer: loadbalancer{
+		Name:         gophercloud.MaybeString(opts.Name),
+		Description:  gophercloud.MaybeString(opts.Description),
+		VipSubnetID:  opts.VipSubnetID,
+		TenantID:     gophercloud.MaybeString(opts.TenantID),
+		VipAddress:   gophercloud.MaybeString(opts.VipAddress),
+		AdminStateUp: opts.AdminStateUp,
+		Flavor:       gophercloud.MaybeString(opts.Flavor),
+		Provider:     gophercloud.MaybeString(opts.Provider),
+	}}
+
+	_, res.Err = c.Post(rootURL(c), reqBody, &res.Body, nil)
+	return res
+}
+
+// Get retrieves a particular virtual IP based on its unique ID.
+func Get(c *gophercloud.ServiceClient, id string) GetResult {
+	var res GetResult
+	_, res.Err = c.Get(resourceURL(c, id), &res.Body, nil)
+	return res
+}
+
+// UpdateOpts contains all the values needed to update an existing virtual
+// Loadbalancer. Attributes not listed here but appear in CreateOpts are
+// immutable and cannot be updated.
+type UpdateOpts struct {
+	// Optional. Human-readable name for the Loadbalancer. Does not have to be unique.
+	Name string
+
+	// Optional. Human-readable description for the Loadbalancer.
+	Description string
+
+	// Optional. The administrative state of the Loadbalancer. A valid value is true (UP)
+	// or false (DOWN).
+	AdminStateUp *bool
+}
+
+// Update is an operation which modifies the attributes of the specified Loadbalancer.
+func Update(c *gophercloud.ServiceClient, id string, opts UpdateOpts) UpdateResult {
+	type loadbalancer struct {
+		Name         *string `json:"name,omitempty"`
+		Description  *string `json:"description,omitempty"`
+		AdminStateUp *bool   `json:"admin_state_up,omitempty"`
+	}
+
+	type request struct {
+		Loadbalancer loadbalancer `json:"loadbalancer"`
+	}
+
+	reqBody := request{Loadbalancer: loadbalancer{
+		Name:         gophercloud.MaybeString(opts.Name),
+		Description:  gophercloud.MaybeString(opts.Description),
+		AdminStateUp: opts.AdminStateUp,
+	}}
+
+	var res UpdateResult
+	_, res.Err = c.Put(resourceURL(c, id), reqBody, &res.Body, &gophercloud.RequestOpts{
+		OkCodes: []int{200, 202},
+	})
+
+	return res
+}
+
+// Delete will permanently delete a particular Loadbalancer based on its unique ID.
+func Delete(c *gophercloud.ServiceClient, id string) DeleteResult {
+	var res DeleteResult
+	_, res.Err = c.Delete(resourceURL(c, id), nil)
+	return res
+}
```

Comment:
can we add the [Show load balancer status tree](http://developer.openstack.org/api-ref-networking-v2-ext.html#showLoadBalancerStatusTree) operation?